src/libLookup/Lookup.cpp

```diff
@@ -1199,7 +1217,11 @@ void Lookup::SendMessageToRandomL2lDataProvider(const bytes& message) const {
   Peer tmpPeer(resolved_ip,
                m_l2lDataProviders[index].second.GetListenPortHost());
   LOG_GENERAL(INFO, "Sending message to l2l: " << tmpPeer);
-  P2PComm::GetInstance().SendMessage(tmpPeer, message);
+  unsigned char startByte = START_BYTE_NORMAL;
```

Comment:
I think we can move this repeating code for sending request into `SendMessage`, so that it will be easier for people who will add new SendMessageToXXX functions.

Reply:
I can move this to `SendMessage` , but it will unnecessary overwrite the `startByte` for response msg as well since we have same `SendMessage` function for both request and response.
let me know if it's ok, otherwise we can discuss on this.

Reply:
Never mind, I think, since it's only in this function that such a call is made.